client/stylesheets/_toCat.scss

```diff
@@ -3947,6 +3950,7 @@ On click the images will be displayed at normal size to complete the effect
 .profileUserCardsUl li img {
     width: 100%;
     height: 100%;
+    border-radius: 4px !important;
```

Comment:
!important should not be used<br>Line should be indented 2 spaces, but was indented 4 spaces